envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,19 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 and HTTP 0.9 requests.
+  // This is off by default, and not fully standards compliant. There is support for pre-HTTP/1.1
```

Comment:
Why is this "not standards compliant"?

Reply:
Documented on https://github.com/envoyproxy/envoy/issues/170, the link I just took out per Matt's request :-)